pkg/scripts/environment.go

```diff
@@ -66,8 +66,15 @@ func ConvertEnvironment(env []Environment) (result []string) {
 
 // ConvertEnvironmentToDocker converts the []Environment into Dockerfile format
 func ConvertEnvironmentToDocker(env []Environment) (result string) {
+	first := true
 	for _, e := range env {
-		result += fmt.Sprintf("ENV %s %s\n", e.Name, e.Value)
+		if first {
+			result += fmt.Sprintf("ENV %s=\"%s\"", e.Name, e.Value)
+			first = false
+		} else {
+			result += fmt.Sprintf(" \\\n\t%s=\"%s\"", e.Name, e.Value)
+		}
 	}
```

Comment:
nice optimization but i'm going to rely on branch prediction, given that adding the length check seems at least equally burdensome in terms of the code complexity.